src/overlays/actors/ovl_En_Dnh/z_en_dnh.h

```diff
@@ -10,7 +10,20 @@ typedef void (*EnDnhActionFunc)(struct EnDnh*, PlayState*);
 typedef struct EnDnh {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnDnhActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x74];
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ u16 unk18C;
+    /* 0x18E */ char pad18E[2];
+    /* 0x190 */ s32 unk190;
+    /* 0x194 */ s32 unk194;
+    /* 0x198 */ s16 unk198;
+    /* 0x19A */ char pad19A[2];
```

Comment:
```suggestion
    /* 0x19A */ UNK_TYPE1 pad19A[2];
```
We use `UNK_TYPE1` for padding instead of `char`